systems-analysis/client-backend-plugin/README.md

```diff
@@ -0,0 +1,77 @@
+# plugin
+
+### Introduction
+
+This document describes the use of a plugin for the Dragonlfy client backend. It instructs developers to use the plugin to enable Dragonfly to support additional backends, such as dfs, without having to break into Dragonlfy to make code changes.
+
+### Usage
+
+The plugin for backend works as a `lib{plugin-name}.so` dynamic link library. Here are the steps to use it:
+
+**1. Plugin put in the specified path**
+
+After the plugin is built, it needs to be stored in the `/usr/local/lib/dragonfly/plugins/dfdaemon/backend` path by default. This can be changed in [dfdaemon config](https://d7y.io/docs/next/reference/configuration/client/dfdaemon/).
```

Comment:
`After the plugin is built, it needs to be put into the /usr/local/lib/dragonfly/plugins/dfdaemon/backend path by default. This can be changed in the [dfdaemon config](https://d7y.io/docs/next/reference/configuration/client/dfdaemon/). Dfdaemon does not support hot reloading of plugins.`